.github/workflows/code-review.yaml

```diff
@@ -52,14 +52,29 @@ jobs:
           echo "COSMOSDB_NAME=${{github.run_id}}" >> $GITHUB_ENV
           echo "COSMOSDB_CONNECTION_STRING=${{secrets.COSMOSDB_CONNECTION_STRING}}" >> $GITHUB_ENV
           echo "MONGODB_NAME=${{github.run_id}}" >> $GITHUB_ENV
+
+      - name: Stop containers if up 
+        run: |
+          docker-compose -f "__integrations__/docker-compose.yml" stop
+          docker-compose -f "__integrations__/docker-compose.yml" down
+          # docker system prune -a -f
       
       - name: Start containers
         run: docker-compose -f "__integrations__/docker-compose.yml" up -d
 
+      - name: Sleep
+        run: sleep 30s
+
```

Comment:
for the moment we can keep this sleep. BTW we should start to think about an healthcheck